tests/playwright/page-object-model/search-results-page.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page, expect } from '@playwright/test'
+
+export class SearchResultsPage {
+  readonly expect: SearchResultsPageAssertions
+  readonly _headerLocator: Locator
+  readonly _searchResultsListLocator: Locator
+
+  constructor (readonly page: Page) {
+    this.expect = new SearchResultsPageAssertions(this)
+    this._headerLocator = this.page.locator('h1')
+    this._searchResultsListLocator = this.page.locator('ul')
```

Comment:
This is ok for now but will need to be changed when we implement the search results page - I've added a task to that story so we don't forget!